src/types.rs

```diff
@@ -664,20 +664,22 @@ impl Rewrite for ast::Ty {
         match self.kind {
             ast::TyKind::TraitObject(ref bounds, tobj_syntax) => {
                 // we have to consider 'dyn' keyword is used or not!!!
-                let is_dyn = tobj_syntax == ast::TraitObjectSyntax::Dyn;
-                // 4 is length of 'dyn '
-                let shape = if is_dyn { shape.offset_left(4)? } else { shape };
+                let shape = match tobj_syntax {
+                    ast::TraitObjectSyntax::Dyn => shape.offset_left(4)?, // 4 is offset 'dyn '
+                    ast::TraitObjectSyntax::DynStar => shape.offset_left(5)?, // 5 is offset 'dyn* '
```

Comment:
hmmm now that I'm thinking about it would it just be better to return `None` if we get to a `DynStart` since its experimental syntax?